src/components/Blog/BlogCarousel/CarouselContainer.tsx

```diff
@@ -104,7 +104,10 @@ export const CarouselContainer = ({
           {title ?? t('blog.recentPosts')}
         </CarouselTitle>
         <CarouselNavigationContainer
-          hide={(itemsCount && itemsCount < 4) || children?.length < 4}
+          show={
```

Comment:
Oh good point! I will be going with the 4!

Btw, it looks odd at first -> But I have not wrapped the logical operator around this component because we always show the "Navigation Arrows" on mobile. So, that "magic" happens within the styled component with breakpoints and is only related to screensizes bigger than mobile.